app/assets/javascripts/discourse/models/post.js

```diff
@@ -336,6 +336,20 @@ const Post = RestModel.extend({
   },
 
   toggleBookmarkWithReminder() {
+    // if we are talking to discobot then any bookmarks should just
+    // be created without reminder options, to streamline the new user
+    // narrative.
+    if (this.username === "discobot" && !this.bookmarked_with_reminder) {
```

Comment:
Hmm I'm wondering if we can override this method in a plugin initializer instead so that discobot specific code can stay within the discobot plugin.